content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+name: tery-hardwicke
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: sam-martin
+        role: editor
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that identifies one group as deservedly superior to another. Common examples are that men are superior to women (sexism) and one ethnicity is superior to all other ethnicities (racism).  
+
+or  
+ 
+> Yes, humans are unique and distinct from other animals in our capacity for complex problem solving. However, pick any species and you will likely find an attribute that also makes them special. 
+ 
+or 
+  
+> No unique set of abilities or characteristics, in itself, provides a criterion for ‘superiority’. 
+For humans to claim that humans are superior because we have specific unique attributes is just another form of prejudice. 
```

Comment:
This line needs `>` at the beginning for it to consistently appear as part of the block quote.